tabbycat/breakqual/aida.py

```diff
@@ -130,8 +130,16 @@ def reinsert_capped_teams(self):
 
 
 @register
-class Aida2016AustralsBreakGenerator(BaseAida2016BreakGenerator):
-    key = "aida-2016-australs"
+class Aida2016Australs3CapBreakGenerator(BaseAida2016BreakGenerator):
+    key = "aida-2016-australs-3cap"
```

Comment:
Sadly, changing this key actually breaks the data in the database: After migration, the value stored in the database will still be `aida-2016-australs`, and Tabbycat will no longer know what that means.

Options:
(1) Leave this as `aida-2016-australs`, or
(2) Write a data migration (as well as a schema migration) to convert this value.

I don't have a strong preference between these.

Reply:
Ahh, got it - Given the old option is still the default for Australs tournaments I would suggest (1); will update!